README.md

```diff
@@ -1,63 +1,33 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Youtube landing page
 
-# Project Name
-
-> One paragraph statement about the project.
+> trying to buile a page that works likeyoutube.
```

Comment:
Fix this grammar error from "buile" to "build" and add a space between the phrase"likeyoutube" to "like youtube"